src/frontend/components/VideoPlayerConnected/VideoPlayerConnected.tsx

```diff
@@ -12,6 +12,7 @@ export const mapStateToProps = (
   state: RootState,
   { video }: VideoPlayerProps,
 ) => ({
+  jwt: state.context.jwt!,
```

Comment:
You can't be sure of this.

Reply:
You can access the video player without jwt token ?

Reply:
No I mean the `!`, you have not done anything to be able to be sure it's non-undefined, non-null

Reply:
You can just drop the `!` and make the `jwt` nullable in the component props

Reply:
done